liferay-echo/liferay-portal
# LPS-147681 Add javax.portlet.version=3.0

Hi @liferay-echo 

it's a follow up of this https://github.com/liferay-lima/liferay-portal/pull/3047#issuecomment-1129761864 original from the issue : https://issues.liferay.com/browse/LPS-153891 and after brian asked if I can solve both issues, the problem on blogs and the problem once solved on blogs and the problem with the panel.


I think I updated the portlet version only on the required places, but can you check if other issues happens.

Steps to Verify

1. Enable the reading time of Blogs : 
Application Menus > Control Panel > System Settings > Blogs > Widget Scope > Check the Enable Reading Time > Save
2. Add a blogs entry (expected result, content visible and editable)
3. Add new Widget Page 
4. Edit new widget page > expected result no error)



Thanks!

cc @adolfopa @ling-alan-huang
